lecture/views.py

```
@@ -0,0 +1,67 @@
+from django.shortcuts import render, redirect, get_object_or_404
```

Ruff (0.8.2)

`django.shortcuts.redirect` imported but unused

Remove unused import: `django.shortcuts.redirect`

(F401)